CHANGELOG.md

```diff
@@ -4,15 +4,23 @@
 - **Breaking:** Remove `linkId` prop from `<Breadcrumb.Item>`, for consistency with other wrapped link components ([#2011])
 - **Breaking:** Remove deprecated functionality from v0.29.x ([#2017])
 - **Breaking:** Remove redundant `caption` prop from `<Carousel.Item>`, as this functionality is available from `<Carousel.Caption>` ([#2018])
+- **Breaking:** Remove pointless `navItem` and `navDropdown` props from `<Button>` ([#2026])
```

Comment:
That would be like a `<Navbar.Button>`, right? Per http://getbootstrap.com/components/#navbar-buttons

The existing support on v0.29.x is just broken though.